environment/workspace/modules/automation/gitops/argocd/update-application/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../../../../manifests/ui
```

Comment:
Should this be `- ui`?

Reply:
This path was required to make kustomizing block show diffs. I changed approach and now it shows the same correct content like in the file which is used for deployment.